### ISSUE TITLE
`quasi_macros` needs updating to 0.16

### ISSUE THREAD

**Issue by petevine** [honeypotio/searchspot#15]
Or it all ends in a failure on nightly:

```shell
Compiling quasi_macros v0.15.0
.cargo/registry/src/github.com-1ecc6299db9ec823/quasi_macros-0.15.0/src/lib.rs:21:5: 21:28 error: function `register` is private 
.cargo/registry/src/github.com-1ecc6299db9ec823/quasi_macros-0.15.0/src/lib.rs:21     quasi_codegen::register(reg);
                                                                                                                        ^~~~~~~~~~~~~~~~~~~~~~~
.cargo/registry/src/github.com-1ecc6299db9ec823/quasi_macros-0.15.0/src/lib.rs:21:29: 21:32 error: mismatched types [E0308]
.cargo/registry/src/github.com-1ecc6299db9ec823/quasi_macros-0.15.0/src/lib.rs:21     quasi_codegen::register(reg);
                                                                                                                                                ^~~
.cargo/registry/src/github.com-1ecc6299db9ec823/quasi_macros-0.15.0/src/lib.rs:21:29: 21:32 help: run `rustc --explain E0308` to see a detailed explanation
/.cargo/registry/src/github.com-1ecc6299db9ec823/quasi_macros-0.15.0/src/lib.rs:21:29: 21:32 note: expected type `&mut syntex::Registry` 
.cargo/registry/src/github.com-1ecc6299db9ec823/quasi_macros-0.15.0/src/lib.rs:21:29: 21:32 note:    found type `&mut rustc_plugin::Registry<'_>` 
error: aborting due to previous error 
error: Could not compile `quasi_macros`.
````

**Comment by RoxasShadow** [comment]
Hello, thanks for reporting this issue. On my end it should be fixed in #14. I'll send a PR to rs-es for doing the same on their hand. I'll get back to you as soon as it is merged.

In the meanwhile you can use Rust stable or adding

```toml
git = "https://github.com/RoxasShadow/rs-es"
branch = "serde-0.7.15"
```

under the rs-es block in `Cargo.toml`.

**Comment by petevine** [comment]
Right, I initially thought it would be enough to update the crate but a full override kept failing too.

**Comment by RoxasShadow** [comment]
#14 compiles on both stable and nightly. It will be merged to master in the next days.

**Comment by RoxasShadow** [comment]
Published as `0.5.0`.

**Comment by petevine** [comment]
Indeed, builds for me now.
